### ISSUE TITLE
Create test models for each core test run

### ISSUE THREAD

**Issue by Hofer-Julian** [Deltares/Ribasim#244]
At the moment, developers have to remember to create the test models for the first checkout and every time something changes in the `testmodels` package. Creating these testmodels takes only about 5 seconds on my machine. Instead of expecting developers to prepare the test models manually, they should simply be created for every test run, ideally in a temporary directory.

The way I envision this code:

1. Check if `ribasim` conda environment is activated, exit with an error if not
2. Create a temporary directory 
3. Setup the models with `ribasim_testmodels` (maybe use https://github.com/cjdoris/PythonCall.jl for that)

**Comment by visr** [comment]
Looks like a duplicate of #116

**Comment by Hofer-Julian** [comment]
Indeed, my bad. Do you mind closing https://github.com/Deltares/Ribasim/issues/116 since its original description is a bit outdated?

**Comment by visr** [comment]
Done. One comment that might still be relevant:

> Let's also use this: https://github.com/cjdoris/CondaPkg.jl

**Comment by Hofer-Julian** [comment]
> Done. One comment that might still be relevant:
> 
> > Let's also use this: https://github.com/cjdoris/CondaPkg.jl

Note: this part needs at least further investigation, but is probably not feasible if `CondaPkg.jl` doesn't allow to activate an existing conda environment.

**Comment by visr** [comment]
Fixed by #604. That doesn't call Python from the Julia tests, but adds `pixi run tests`, which generate the test models first. So I'd say this is fixed if you just run `pixi run tests`.

Running `pkg> test` is probably still, but then you need to remember to run `pixi run generate-testmodels` first yourself.

```
tests = { depends_on = [
    "lint",
    "generate-testmodels",
    "test-ribasim-python",
    "test-ribasim-core",
] }
```
